### ISSUE TITLE
mitochondrial genome 

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#2497]
From Li-Lin:

mitochondrial genomes, and while doing so, found some reference sequence errors in the mitochondrial genome and the K-region.

The paper was published ahead of print last week.
https://www.ncbi.nlm.nih.gov/pubmed/31364709
https://academic.oup.com/gbe/advance-article/doi/10.1093/gbe/evz165/5541790

The mitochondrial genome sequence of the reference strain has been deposited at:
https://www.ncbi.nlm.nih.gov/nuccore/MK618072.1/

The K-region sequence of the reference strain (without the repetitive cenH region) has been deposited at:
https://www.ncbi.nlm.nih.gov/nuccore/MK618141.1/

**Comment by ValWood** [comment]
I updated to this version of the mitochondrial genome.

I copied across our existing annotation:
GO
products
status 
GO
species distribution
warnings
orthologs
gene expression.

I added the anticodons to the tRNA products

I deleted the gene features and the exon features  and translations as we don't explicitly annotate these (can be inferred)

Updated  some products.

kept protein IDs

Committed revision 6244.
